test/unit/org/pih/warehouse/utils/ConfigHelperTests.groovy

```diff
@@ -0,0 +1,67 @@
+package org.pih.warehouse.utils
+
+import grails.test.GrailsUnitTestCase
+import org.junit.Test
+import org.pih.warehouse.core.RoleType
+import util.ConfigHelper
+
+class ConfigHelperTests extends GrailsUnitTestCase {
+    @Test
+    void findAccessRule_shouldReturnCorrectRule() {
```

Comment:
This is awesome that you added tests for this. We're going to want to write the tests in Spock if possible. I'm not sure how to 

With that said, you can add configuration properties to a Junit test like this
 
```class ConfigHelperTests extends GrailsUnitTestCase {
    @Before
    void setUp() {
        super.setUp()
        mockConfig("openboxes.anonymize.enabled = false")
    }
```
Might be more complicated for this but I think we can just use Groovy DSL. The following might not be correct so you might have to massage it to get it to work.
```
String rbacRulesConfig = """
openboxes { 
    security {
        rbac { 
            rules {
                controller = "order"
                action = "remove"
                accessRules: [ minimumRequiredRole: RoleType.ROLE_MANAGER ]
            }
        }    
    }    
}
"""
mockConfig(rbacRulesConfig)
```
The other option would just be to copy the code 
```
String rbacRulesConfig = """
[
  [controller: 'order', actions: ['remove'], accessRules: [ minimumRequiredRole: RoleType.ROLE_MANAGER ]],
  [controller: 'order', actions: ['removeOrderItem'], accessRules: [ minimumRequiredRole: RoleType.ROLE_MANAGER ]],
]
```